nuxeo-document-history-page-provider.html

```diff
@@ -36,14 +36,15 @@
         display: none;
       }
     </style>
-
-    <nuxeo-operation id="auditOp"
-      op="Audit.QueryWithPageProvider"
-      enrichers="{{enrichers}}"
-      schemas="[[schemas]]"
-      headers="{{headers}}">
-    </nuxeo-operation>
-
+    <nuxeo-audit-page-provider id="auditOp" provider-name="DOCUMENT_HISTORY_PROVIDER"
+                               auto="[[auto]]"  auto-delay="[[autoDelay]]"
```

Comment:
boolean attributes need to rely on `auto$="[[auto]]"` because it only checks whether the attribute is there or not, meaning `auto="false"` would still set `auto` to `true`